src/components/Navbar.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from "react";
+
+import Brand from './Brand.jsx';
+import Logo from './Logo.jsx';
+import Navbutton from './Navbutton.jsx';
+
+class NavBar extends Component { 
+  cunstructor(props) {
```

Comment:
I see your point. My thinking was that the navbar would most likely have functionality that requires state. However I realize thats an untested assumption. I'm ok to refactor on the next commit. kinda annoying to do on github.